assets/react/v3/entries/course-builder/components/curriculum/Topic.tsx

```diff
@@ -206,10 +206,14 @@ const Topic = ({ topic, onDelete, onCopy, onSort, onCollapse, isOverlay = false
               triggerRef={deleteRef}
               closePopover={() => setIsDeletePopoverOpen(false)}
               maxWidth="258px"
-              title={`Delete topic "${topic.post_title}"`}
-              message="Are you sure you want to delete this content from your course? This cannot be undone."
+              title={__('Delete topic ', 'tutor') + `"${topic.post_title}"`}
+              message={__(
+                'Are you sure you want to delete this content from your course? This cannot be undone.',
+                'tutor'
+              )}
               animationType={AnimationType.slideUp}
-              arrow="auto"
+              arrow="top"
```

Comment:
The popover will go outside of the viewport if it is at the bottom of the screen.